src/common/dto/sessions/response/get.session.response.ts

```
@@ -1,5 +1,7 @@
 import { Session } from '@prisma/client';
 
+type SessionWithUser = Session & { user: User };
```

ESLint

[error] 'SessionWithUser' is defined but never used.

(@typescript-eslint/no-unused-vars)